api/users/urls.py

```diff
@@ -6,6 +6,6 @@
     url(r'^$', views.UserList.as_view(), name=views.UserList.view_name),
     url(r'^(?P<user_id>\w+)/$', views.UserDetail.as_view(), name=views.UserDetail.view_name),
     url(r'^(?P<user_id>\w+)/nodes/$', views.UserNodes.as_view(), name=views.UserNodes.view_name),
-    url(r'^(?P<user_id>\w+)/registrations/$', views.UserRegistrations.as_view(), name=views.UserRegistrations.view_name)
-
+    url(r'^(?P<user_id>\w+)/node_logs/$', views.UserNodeLogs.as_view(), name=views.UserNodeLogs.view_name),
```

Comment:
That was something that I thought would imply that the logs belong to the user, which is not accurate, as they both contain logs created by other users, and not contain logs attached to a node that the user is no longer a contributor of.